src/main/java/com/redhat/cloud/notifications/RestValidationClient.java

```diff
@@ -0,0 +1,23 @@
+package com.redhat.cloud.notifications;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+@Path("/internal/validation")
+@RegisterRestClient(configKey = "notifications-backend")
+public interface RestValidationClient {
+
+    @GET
+    @Path("/baet")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.APPLICATION_JSON)
```

Comment:
changed. Why isn't this necessary? Is json the default?

Reply:
It is not needed because you don't expect a body, so there is no hint needed on the body encoding (i.e. the caller does not need to supply a valid/matching content-type header)